<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright (C) 2006 Peter Simonsson &lt;psn@linux.se&gt;</author>
 <comment>This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2, or (at your option) any later version.

This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
</comment>
 <class>SearchBarBase</class>
 <widget class="QWidget" name="SearchBarBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>28</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="m_closeButton">
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_findLabel">
       <property name="text">
        <string>F&amp;ind:</string>
       </property>
       <property name="buddy">
        <cstring>m_searchEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="m_searchEdit"/>
     </item>
     <item>
      <widget class="KPushButton" name="m_findNextButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_findPreviousButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Previous</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_optionsButton">
       <property name="text">
        <string>&amp;Options</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="4"/>
 <layoutfunction spacing="KDialog::spacingHint"/>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
